src/utils/e2ee/encryption.js

```diff
@@ -500,7 +500,7 @@ class Encryption {
 			const key = base64js.toByteArray(decoded.key)
 			const index = decoded.index
 
-			if (!isEqual(sessionData.lastKey, key)) {
+			if (!isEqualJson(sessionData.lastKey, key)) {
```

Comment:
Wasn't sure, but tested. Keys are `Uint8Array`, and `base64js.toByteArray` returns `Array`. So should be fine to use it here?

![image](https://github.com/user-attachments/assets/ca17d0c2-f2d4-4820-997a-2bc1b8fb8e94)